Libs/MRML/Widgets/qMRMLPlotView.cxx

```diff
@@ -1041,6 +1064,10 @@ void qMRMLPlotView::keyPressEvent(QKeyEvent *event)
     {
     d->RecalculateBounds();
     }
+  if (event->key() == Qt::Key_D)
```

Comment:
I would think that ```D``` means delete selected point(s). Maybe we could use ```U``` - unselect instead?

Reply:
ok